pycbc/waveform/generator.py

```diff
@@ -38,6 +38,11 @@
 #
 
 # utility functions/class
+def _default_pregenerate(*args):
+    pass
+
+def _default_postgenerate(res):
```

Comment:
@ahnitz What does this mean? Does it expect 2 blank lines between each function? Or after the function definition? Is it getting confused because there's no doc string for these functions?